### ISSUE TITLE
Package full build of fluent-web.js on npm / build command is failing

### ISSUE THREAD

**Issue by meandavejustice** [projectfluent/fluent-web#6]
npm installation doesn't include the built file `fluent-web.js`

![ls-fluent](https://user-images.githubusercontent.com/1844554/50619180-ae78a880-0ec5-11e9-81d5-de404b4ff291.png)

Unfortunately I can't just build it myself locally (may be cus I'm running `npm start` inside of [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) but I usually don't run into issues like this.

Here is the error I get in my console:
```
meandave@LAPTOP-NA0Q0G86:/mnt/c/Users/Dave Justice/Code/fluent-web$ npm start

> fluent-web@0.0.1 start /mnt/c/Users/Dave Justice/Code/fluent-web
> make -f makefile build & npx http-server ./examples

fs.js:1657
      binding.lstat(baseLong);
              ^

Error: ENOENT: no such file or directory, lstat '/mnt/c/Users/Dave'
    at Object.realpathSync (fs.js:1657:15)
    at runRollup (/mnt/c/Users/Dave Justice/Code/fluent-web/node_modules/rollup/bin/rollup:1946:29)
    at Object.<anonymous> (/mnt/c/Users/Dave Justice/Code/fluent-web/node_modules/rollup/bin/rollup:2010:5)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
makefile:9: recipe for target 'fluent-web.js' failed
make: *** [fluent-web.js] Error 1
Starting up http-server, serving ./examples
Available on:
  http://169.254.82.6:8080
  http://127.0.0.1:8080
  http://192.168.1.2:8080
Hit CTRL-C to stop the server
```

In the browser I'm seeing the same error as mentioned in #2 
```
SyntaxError: import declarations may only appear at top level of a module
```

**Comment by meandavejustice** [comment]
Need to get this working this week for a testpilot related thing, If you don't get a chance to publish soon would you mind posting a link/paste/gist of the built file? Thank You :)

**Comment by zbraniecki** [comment]
Just published fluent-web  0.1.1 which should fix this issue! https://www.npmjs.com/package/fluent-web please test :)

**Comment by meandavejustice** [comment]
@zbraniecki Thanks it's working great, also the examples are working for me now, so you can probably close #2 too
